config/database.yml

```diff
@@ -0,0 +1,15 @@
+development:
+  adapter: postgresql
+  encoding: utf8
+  database: project_development
```

Comment:
this mean that I need the same databse name, change it to <%= ENV["WIZZART_DATABABASE %> and set env variable on your pc